File: 26. Remove Duplicates from Sorted Array.c
int removeDuplicates(int* nums, int numsSize) {
    if(numsSize==0)
        return 0;
    
    int ret=1;
    int c = nums[0];

    for(int i=0; i<numsSize; i++){
        #printf("nums[%d]=%d\n", i, nums[i]);
        if(nums[i] != c){
            #printf("found!\n");
            c = nums[i];
            ret+=1;
            nums[ret-1] = c;
        }
    }    
    
    #printf("ret=%d\n", ret);
    
    return ret;
}

File: 26. Remove Duplicates from Sorted Array.py
class Solution:
    def removeDuplicates(self, nums: 'List[int]') -> 'int':
        print('input:', nums)
        
        if len(nums) == 0:
            return 0
        
        
        c=nums[0]
        ret = 1
        for n in nums:
            if n==c:
                pass
            else:
                c=n
                ret+=1
                nums[ret-1] = c
        
        print('len=', ret)
        return ret 
